server.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const cors = require('cors');
+const app = express();
+  var bodyParser = require('body-parser');
+
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('./knexfile')[environment];
+const database = require('knex')(configuration);
+
+app.use(bodyParser.urlencoded({
+    extended: true
+}));
+app.use(bodyParser.json());
+app.use(cors());
```

Comment:
3. Setup bodyParser to parse incoming request bodies in a middleware and cors to allow access to server from another domain